app/boot.js

```diff
@@ -5,6 +5,7 @@ var config = require("../config/config"),
 
 module.exports = {
   es: new elasticsearch.Client({
+    apiVersion: "1.7",
```

Comment:
Well the present server is running 1.3.2, so it's compatible enough for our purposes